README.md

```diff
@@ -22,6 +22,24 @@ Alternatively binaries are available from SourceForge at <http://sourceforge.net
 You are encouraged to review the release notes ([releasenotes.txt](releasenotes.txt)), particularly when upgrading to a 
 later version. The release notes will generally document any breaking changes.
 
+Nightly Development Builds
+--------------------------
+
+The quickest way to get the latest release of NHibernate is to add it to your project using 
```

Comment:
I'm not sure we should call it latest release.  I think It's more appropriate to call it latest build.

P.S. We should also add link to nightly builds to nhibernate.info  and maybe to nuget package details

Reply:
Yeah, was a copy-paste.

Reply:
Changed